packages/forms/resources/views/components/file-upload.blade.php

```diff
@@ -142,6 +139,13 @@
             />
         </div>
 
+        <div
```

Comment:
Any reason why you're using `<div>` here instead of `<p>`, @danharrin?
Also, why not just reuse the existing `error-message` Blade component?

Reply:
`p` would work, not using the Blade component as this is not the field wrapper

Reply:
Perhaps extract the common code from the field wrapper error message and this file then, because it needs to be themeable using CSS hook classes as well.